pages/your_domain/structural_bioinformatics.md

```diff
@@ -48,3 +48,19 @@ Researchers in the field should be able to find predictions of macromolecular st
+
+## Storing and sharing experimentally solved structures
+ 
+### Description
+
+Experimentally solved atomic structures of molecules can be obtained by several methods, such as X-ray crystallography,  Nuclear Magnetic Resonance (NMR) spectroscopy, and 3D Electron Microscopy. Here you can find useful tools and guides for storing and sharing structure models based on these methods.
+* Structure models resulting from experimental methods are broadly in the {% tool "pdb" %} under the [CC0 1.0 Universal (CC0 1.0) Public Domain Dedication](https://creativecommons.org/publicdomain/zero/1.0/).
+* Additionally, raw and intermediate data associated with the structure model can be also published in different curated data archives depending on the methods used. Raw EM data and processed 3D volumes and tomograms can be stored in {% tool "empiar" %} and {% tool "emdb" %}, respectively. Raw data from NMR studies can be stored in {% tool "bmrb" %}. For X-ray diffraction experiments raw data can be stored in {% tool "irrmc" %} and {% tool "sbgrid-data-bank" %}. Data submitted in these repositories can be cross-referenced to related PDB entries.
+* To extend molecular context, structure models can be visualized and analyzed along with respective volume maps and {% tool "uniprot" %} sequences using data aggregators such as {% tool "3dbionotes" %}, either if they have been published in PDB and EMDB or if not. A bunch of biochemical, biomedical and validation annotations would be mapped on the structure model coordinates and sequence to let the user better understand macromolecular function and interactions. Users can also use the [COVID-19 Structural Hub](https://3dbionotes.cnb.csic.es/ws/covid19), a dedicated summary view for all published SARS-CoV-2 structure models. 
+* As well as for computationally solved structures, model coordinates and metadata for experimentally solved structures are also preferably stored using the standard PDB archive format {% tool "pdbx-mmcif-format-and-tools" %} and the {% tool "pdbx-mmcif-modelcif-extension-dictionary" %}, respectively.
+* Data model and metadata standards for submitting data underpinning macromolecular models depend on the experimental method used. EMDB map distribution format description has broadly followed [CCP4](https://www.ccp4.ac.uk/html/maplib.html) and [MRC](https://www2.mrc-lmb.cam.ac.uk/research/locally-developed-software/image-processing-software/) map format. Metadata is contained in a header file, an XML file that follows the XSD data model. EMPIAR data model schema consists of the main empiar.xsd XML schema file and additional requirements in empiar.sch in Schematron format (see [here](ftp://ftp.ebi.ac.uk/pub/databases/emtest/empiar/schema/) for more details). BMRB (meta)data distribution format is based on NMR-STAR, an extension of the Self-defining Text Archive and Retrieval (STAR) file format.
```

Comment:
I would not link to an ft[ address since most browsers can't open it ...

Reply:
I have replace the ftp link for other related https link they provide in the paper (https://ftp.ebi.ac.uk/pub/databases/emtest/empiar/schema/). Would it be better this way?